include/app.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <aos/common/error.hpp>
+
+/**
+ * Aos zephyr application.
+ */
+class App {
+public:
+    /**
+     * Initializes application.
+     *
+     * @return aos::Error
+     */
+    aos::Error Init();
+    /**
+     * Returns Aos application instance.
+     * @return App&
+     */
+
+    static App& Get()
```

Comment:
If we use the singleton pattern in the class, I recommend deleting the copy operation

Reply:
agree